cmd/dns_records_add.go

```diff
@@ -109,13 +109,75 @@ func init() {
 	dnsAddCmd.AddCommand(dnsAAAACmd)
 	dnsAAAACmd.Flags().StringP("name", "n", "", "Leave this blank to create a record for <domain name>, You may use the '*' wildcard here.")
 	dnsAAAACmd.Flags().StringP("address", "a", "", "Example: 2001:0db8:85a3:0000:0000:EA75:1337:BEEF")
-	dnsAAAACmd.Flags().IntP("ttl", "t", 3600, "The time in second to leave (refresh rate) of the record.")
+	dnsAAAACmd.Flags().IntP("ttl", "t", 3600, "The time in seconds to live (refresh rate) of the record.")
 
 	if err := dnsAAAACmd.MarkFlagRequired("address"); err != nil {
 		log.Fatal(err)
 	}
 }
 
+// dnsCAACmd represents the URL command
+var dnsCAACmd = &cobra.Command{
+	Use:   "CAA <domain name>",
+	Short: "Add CAA record type to a domain",
+	Long: `A Certification Authority Authorization (CAA) record is used to specify which certificate
+authorities (CAs) are allowed to issue certificates for a domain.
+
+What is a flag? <https://tools.ietf.org/html/rfc6844#section-3>
+	`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
```

Comment:
Maybe `!= 1` ?

Reply:
It's not mandatory a the moment, because in other exo cmd it's like that for non fixed parameters and fixed parameters. So maybe another PR to refacto that. thank you